core/templates/components/forms/validators/is-valid-url.filter.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
We should have a pipe for this as well like all other angular filters.

@bansalnitish can you please confirm?

Reply:
Yes you're right @ankita240796